patroni/postgresql.py

```diff
@@ -1006,9 +1006,10 @@ def clone(self, clone_member):
     def bootstrap(self, config):
         """ Initialize a new node from scratch and start it. """
         if self._initialize(config) and self.start() and self.run_bootstrap_post_init(config):
-            for name, value in config['users'].items():
-                if name not in (self._superuser.get('username'), self._replication['username']):
-                    self.create_or_update_role(name, value['password'], value.get('options', []))
+            if 'users' in config:
```

Comment:
It will fail if the users key is empty. I propose the following one-liner instead:
```
diff --git a/patroni/postgresql.py b/patroni/postgresql.py
index 814fb6c..332a99d 100644
--- a/patroni/postgresql.py
+++ b/patroni/postgresql.py
@@ -1006,7 +1006,7 @@ $$""".format(name, ' '.join(options)), name, password, password)
     def bootstrap(self, config):
         """ Initialize a new node from scratch and start it. """
         if self._initialize(config) and self.start() and self.run_bootstrap_post_init(config):
-            for name, value in config['users'].items():
+            for name, value in (config.get('users') or {}).items():
                 if name not in (self._superuser.get('username'), self._replication['username']):
                     self.create_or_update_role(name, value['password'], value.get('options', []))
             self.create_or_update_role(self._replication['username'], self._replication['password'], ['REPLICATION'])
```